Firestore/core/include/firebase/firestore/firestore_errors.h

```diff
@@ -23,14 +23,11 @@ namespace firestore {
 /**
  * Error codes used by Cloud Firestore.
  *
- * The codes are in sync with the Firestore iOS client SDK header file
- * FIRFirestoreErrors.h.
+ * The codes are in sync across Firestore SDKs on various platforms.
```

Comment:
This file is part of the public headers across platforms, so I rephrased this comment not to be iOS-centric.